scripts/licenses/csv.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/csv"
+	"go/build"
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/google/trillian/scripts/licenses/licenses"
+	"github.com/spf13/cobra"
+)
+
+var (
+	gitRemotes []string
+
+	csvCmd = &cobra.Command{
+		Use:   "csv <package>",
+		Short: "Prints all licenses that apply to a Go package and its dependencies",
+		Args:  cobra.ExactArgs(1),
+		RunE:  csvMain,
+	}
+)
+
+func init() {
+	csvCmd.Flags().StringArrayVar(&gitRemotes, "git_remote", []string{"origin"}, "Remote Git repositories to try")
```

Comment:
I use `upstream` a lot, dunno if it's common enough to add as a default here.

Reply:
I've tweaked the error logging so it'll be quieter when trying different remotes, so long as one of them succeeds. That makes having extra values in here less noisy, so I've added "upstream" since it seems like it's generic enough that it may be common.